tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecPullRequestTools/SpecPullRequestTools.cs

```diff
@@ -134,12 +134,15 @@ public async Task<List<string>> CreatePullRequest(string title, string descripti
 
                     //Get repo details like target owner, head owner, repo name
                     var headRepoOwner = await gitHelper.GetRepoOwnerNameAsync(repoRootPath, false);
+                    var targetRepoOwner = await gitHelper.GetRepoOwnerNameAsync(repoRootPath, true);
```

Comment:
@jeo02  late to the party, but I wonder if this method, `GetPullRequestForCurrentBranch` , and other generic methods should be moved out of `SpecPullRequestTools` and into a common place. that way any repo can beneficiate from them.
When we have specific logic for the specs repo, then we can leave here, etc...